pkg/sessionctx/context.go

```diff
@@ -86,89 +78,33 @@ type InstancePlanCache interface {
 
 // Context is an interface for transaction and executive args environment.
 type Context interface {
-	SessionStatesHandler
-	contextutil.ValueStoreContext
+	planctx.Common
```

Comment:
Do we need `planctx.Common` here? A user can call `Context. GetPlanCtx()`

Reply:
`sessionctx.Context` need those API defined in `planctx.Common` which are shared with `PlanCtx`, i just unify the definition of those APIs